### ISSUE TITLE
hugo version v0.97.0 rendering error

### ISSUE THREAD

**Issue by Y4er** [razonyang/hugo-theme-bootstrap#505]
**Describe the bug**
hugo version v0.97.0 rendering error.

**Screenshots**
![image](https://user-images.githubusercontent.com/40487319/163547522-29ef1a8f-0fb6-4d9b-94b5-10232267fdb8.png)

**Hugo version:**

```shell
$ hugo version
```
v0.97.0 
**Theme version:**
0.75.1

**Comment by razonyang** [comment]
I guess this is caused by the `postcss` which will remove most of the CSS if the following configuration is not set.

```toml
[build]
  writeStats = true
```

> It tells Hugo to generate the `hugo_stat.json` file, used by `postcss` for purging unused CSS.

**Comment by Y4er** [comment]
![image](https://user-images.githubusercontent.com/40487319/163655252-c2024106-74fb-4bca-b4db-bbad24e54f71.png)

actually,i have set this.

**Comment by Y4er** [comment]
in my localhost:1313

hugo version 0.96.0 html is rendered as
```
<link rel="stylesheet" href="http://localhost:1313/assets/main/bundle.min.87fc6e639c81cc56a4ccd268ca277524d1890d44f26c83aa95dc38d2319adfe9.css" integrity="sha256-h/xuY5yBzFakzNJoyid1JNGJDUTybIOqldw40jGa3+k="
```

hugo version 0.97.0 html is rendered as
```
<link rel="stylesheet" href="[__h_pp_l1_7_Permalink__e=](http://localhost:1313/__h_pp_l1_7_Permalink__e=)" integrity="__h_pp_l1_7_Data.Integrity__e=" crossorigin="anonymous">
```

i don't know why

**Comment by razonyang** [comment]
Just upgrade to `0.97.0` and reproduced this issue.

**Comment by razonyang** [comment]
It should be a bug or breaking change of Hugo `0.97.0`. You'll need to execute `hugo server` with the `--renderToDisk` flag.

**Comment by razonyang** [comment]
Confirmed that this has been fixed by hugo `0.97.1`.
